libretroshare/src/pqi/p3netmgr.cc

```diff
@@ -745,24 +760,35 @@ void p3NetMgrIMPL::netExtCheck()
 			if (mUseExtAddrFinder)
 			{
 #if defined(NETMGR_DEBUG_TICK) || defined(NETMGR_DEBUG_RESET)
-				std::cerr << "p3NetMgrIMPL::netExtCheck() checking ExtAddrFinder" << std::endl;
+				RS_DBG("checking ExtAddrFinder");
 #endif
-                sockaddr_storage tmpip = mLocalAddr;	// copies local port and correctly inits the IP family
-
-                bool extFinderOk = mExtAddrFinder->hasValidIP(tmpip);
+				sockaddr_storage tmpip = mLocalAddr;	// copies local port and correctly inits the IP family
```

Comment:
Done
Should we override = operator and make compil failed or at least mark it as deprecated?

Reply:
Making compilation fail is the right thing to do, but how we do ensure our override is visible on all RS code? But of course we could at least make it visible everywhere rsnet.h is included. Anyway be prepared to lot of compiler errors, because I guess for `sockaddr_storage =` is used a bunch of time around the code, so better having a distinct PR just for that.